### DISCUSSION TITLE
bro plzzzzzz make tokken genrator for discord plz

### DISCUSSION THREAD

**Discussion by NOFY96** [community/community#31015] [Other Feature Feedback, Questions, & Ideas]
plzzzzzzzzzzzzzzzzzzzzzzzzz make discord token genraotr

**Comment by AmineZCS** [comment]
what does a discord token do ?

**Reply by KingTino104** [reply]
discord token = discord cookie

**Reply by NOFY96** [reply]
> discord token = discord cookie

yse i know

**Reply by NOFY96** [reply]
> what does a discord token do ?

discord tokken is discord  cooki
